spring-test/src/test/java/org/springframework/test/web/client/samples/SampleTests.java

```diff
@@ -155,24 +152,19 @@ public void performGetWithResponseBodyFromFile() {
 	public void verify() {
 
 		this.mockServer.expect(requestTo("/number")).andExpect(method(HttpMethod.GET))
-			.andRespond(withSuccess("1", MediaType.TEXT_PLAIN));
+				.andRespond(withSuccess("1", MediaType.TEXT_PLAIN));
 
 		this.mockServer.expect(requestTo("/number")).andExpect(method(HttpMethod.GET))
-			.andRespond(withSuccess("2", MediaType.TEXT_PLAIN));
+				.andRespond(withSuccess("2", MediaType.TEXT_PLAIN));
 
 		this.mockServer.expect(requestTo("/number")).andExpect(method(HttpMethod.GET))
-			.andRespond(withSuccess("4", MediaType.TEXT_PLAIN));
+				.andRespond(withSuccess("4", MediaType.TEXT_PLAIN));
 
 		this.mockServer.expect(requestTo("/number")).andExpect(method(HttpMethod.GET))
-			.andRespond(withSuccess("8", MediaType.TEXT_PLAIN));
-
-		@SuppressWarnings("unused")
-		String result1 = this.restTemplate.getForObject("/number", String.class);
-		// result1 == "1"
+				.andRespond(withSuccess("8", MediaType.TEXT_PLAIN));
 
-		@SuppressWarnings("unused")
-		String result2 = this.restTemplate.getForObject("/number", String.class);
-		// result == "2"
+		assertThat(this.restTemplate.getForObject("/number", String.class)).isEqualTo("1");
```

Comment:
this is some real benefit, imho. Testing the real assertion rather commenting about it